src/unix/notbsd/linux/other/mod.rs

```diff
@@ -321,7 +321,6 @@ pub const SOL_PNPIPE: ::c_int = 275;
 pub const SOL_RDS: ::c_int = 276;
 pub const SOL_IUCV: ::c_int = 277;
 pub const SOL_CAIF: ::c_int = 278;
-pub const SOL_ALG: ::c_int = 279;
```

Comment:
Why is this constant being removed?

Reply:
The idea is to support AF_ALG related constants for both Linux and Android.

Reply:
I guess what I am asking is, this constant was in `notbsd/linux/other/mod`, and now it is in `notbsd/mod.rs`. Have you checked that the new value is correct for `emscripten`, `musl`, etc. ?

Reply:
[x] emscripten - https://github.com/emscripten-core/emscripten/blob/6e4b98636618989bcd99308391e51aa1b81f4c61/system/include/libc/sys/socket.h#L253
[x] musl https://git.musl-libc.org/cgit/musl/tree/include/sys/socket.h#n266
[x] android https://github.com/aosp-mirror/platform_bionic/blob/3e5a32ce2c24969932634d84e2755755094fa582/libc/include/sys/socket.h#L288

I think it  doesn't make any sense for `emscripten`. What it the best location for constants applicable only for Linux and Android?

Reply:
Thanks, LGTM. 

> I think it doesn't make any sense for `emscripten`. What it the best location for constants applicable only for Linux and Android?

Add them independently to `linux/mod.rs` and `android/mod.rs`, but in this case since they are also available in `emscripten` (even though they are probably useless), adding them to the `notbsd/mod.rs` module is fine.